ethereumj-core/src/main/java/org/ethereum/net/rlpx/MessageCodesResolver.java

```diff
@@ -34,12 +36,21 @@ public void init(List<Capability> caps) {
         int offset = P2pMessageCodes.USER.asByte() + 1;
 
         for (Capability capability : caps) {
+
             if (capability.getName().equals(Capability.ETH)) {
                 setEthOffset(offset);
                 EthVersion v = fromCode(capability.getVersion());
                 offset += EthMessageCodes.values(v).length;
             }
 
+            if (capability.getName().equals(Capability.PAR)) {
+                // TODO: Why???
+                setParOffset(33);
```

Comment:
Hmm, yep, bad hardcode :/ 
But it seems there is nothing we can do here
Parity doesn't support other protocols like SHH and BZZ so it seems they don't care much about this